### PR TITLE
Add missing python endpoint to the quickstart

### PR DESCRIPTION
**Pull request by lbalmaceda**

### CODE REVIEW

File: snippets/server-apis/python/use.md [hunk 1]
```diff
@@ -2,17 +2,30 @@
...
+@APP.route("/api/public")
+@cross_origin(headers=["Content-Type", "Authorization"])
 def public():
     response = "Hello from a public endpoint! You don't need to be authenticated to see this."
     return jsonify(message=response)
 
 # This does need authentication
-@app.route("/api/private")
-@cross_origin(headers=['Content-Type', 'Authorization'])
+@APP.route("/api/private")
```
Comment (lbalmaceda) [inline snippets/server-apis/python/use.md:12]: This should be uppercase

**Comment by lbalmaceda** [comment]
@alexisluque was this left a side on purpose? I find the next step of the quickstart already consuming that endpoint.

**Comment by alexisluque** [comment]
@lbalmaceda It wasn't left a side on propouse. The endpoint `/api/private-scoped` should be on the Quickstart.

File: snippets/server-apis/python/use.md [hunk 2]
```diff
@@ -2,17 +2,30 @@
...
+@cross_origin(headers=["Content-Type", "Authorization"])
 @requires_auth
 def private():
     response = "Hello from a private endpoint! You need to be authenticated to see this."
     return jsonify(message=response)
+
+# This does need authorization
```
Comment (damieng) [inline snippets/server-apis/python/use.md]: Would could just be "This needs authorization".  I immediately saw "doesn't" when reading FWIW.

**Review by damieng — DISMISSED** [review]
One minor suggestion, up to you if you want to change it or not.

File: snippets/server-apis/python/use.md (hunk 2 above)
Comment (lbalmaceda) [reply snippets/server-apis/python/use.md]: Absolutely.

**Review by damieng — APPROVED** [review]

**Pull request merged**